src/interpreter.rs

```diff
@@ -86,16 +86,15 @@ struct LoopExpr {
 }
 
 impl Interpreter {
-    pub fn new(modules: &[Ref<Module>]) -> Result<Interpreter> {
+    pub fn new() -> Result<Interpreter> {
```

Comment:
In future we can change this so that Interpreter is returned instead of Result<Interpreter>